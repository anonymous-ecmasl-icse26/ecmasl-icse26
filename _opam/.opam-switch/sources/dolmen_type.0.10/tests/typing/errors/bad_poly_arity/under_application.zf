
val f : pi a b. a -> b -> prop.

goal f int 5 6.

