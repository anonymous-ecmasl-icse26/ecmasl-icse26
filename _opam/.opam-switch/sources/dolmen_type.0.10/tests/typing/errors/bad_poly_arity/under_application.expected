File "tests/typing/errors/bad_poly_arity/under_application.zf", line 4, character 5-14:
4 | goal f int 5 6.
         ^^^^^^^^^
Error This function expected exactly 2 type arguments, since term arguments
      are also provided, but was given 1 arguments.
